Monika-After-Story/MonikaModDev
# Heart Choker

#6020 - adds heart choker by @SirNimblyBottoms

**NOTE:** SirNimbly made the JSON and sprites, I have just checked that it shows up in game. Make edits to select dialogue or a specific reaction if desired.

# Testing
* Verify spritepack loads
* Verify choker looks good
